protocol/attestation.go

```diff
@@ -146,10 +149,44 @@ func (attestationObject *AttestationObject) Verify(relyingPartyID string, client
 	// Step 14. Verify that attStmt is a correct attestation statement, conveying a valid attestation signature, by using
 	// the attestation statement format fmt’s verification procedure given attStmt, authData and the hash of the serialized
 	// client data computed in step 7.
-	attestationType, _, err := formatHandler(*attestationObject, clientDataHash)
+	attestationType, x5c, err := formatHandler(*attestationObject, clientDataHash)
```

Comment:
What exactly does this break? The way metadata usage is designed here, the x5c is only evaluated in the general attestation parser when the aaguid is found in metadata, otherwise it should be skipped.

Reply:
This PR adds unit tests for Apple as well.